tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -1633,6 +1722,49 @@ class SomeClass {}',
 function foo($v) {}',
         ];
 
+        yield 'Test PHPDoc union with imports' => [
+            '<?php
+
+namespace Ns;
+use Other\Foo;
+use Other\FooŇ;
+
+/**
+ * @param \Exception|\Exception2|int|Foo|FooŇ|null $v
+ */
+function foo($v) {}',
+            '<?php
+
+namespace Ns;
+
+/**
+ * @param \Exception|\Exception2|int|\Other\Foo|\Other\FooŇ|null $v
+ */
+function foo($v) {}',
+            ['import_symbols' => true],
+        ];
+
+        yield 'Test PHPDoc string must be kept as is' => [
+            '<?php
+
+namespace Ns;
+use Other\Foo;
+
+/**
+ * @param Foo|\'\Other\Bar|\Other\Bar2|\Other\Bar3\'|\Other\Foo2 $v
```

Comment:
Shouldn't it be like this?

```suggestion
use Other\Foo;
use Other\Foo2;

/**
 * @param Foo|\'\Other\Bar|\Other\Bar2|\Other\Bar3\'|Foo2 $v
```

I understand that string pseudo-type is untouched, but what's stopping the second proper FQCN from being imported?

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7619#issuecomment-1890931444 - in this PR, we fix issues only